analyzer.py

```diff
@@ -8,11 +8,15 @@
 from propertysuggester.utils.CompressedFileType import CompressedFileType
 
 if __name__ == "__main__":
-    logging.basicConfig(level=20) # Print logging.info
+    logging.basicConfig(level=20)  # Print logging.info
 
-    parser = argparse.ArgumentParser(description="this program generates a correlation-table from the csv-dump")
-    parser.add_argument("input", help="The CSV input file (wikidata triple)", type=CompressedFileType('rb'))
-    parser.add_argument("output", help="The CSV output file (default=sys.stdout)", default=sys.stdout, nargs='?',
+    parser = argparse.ArgumentParser(
+        description="this program generates a correlation-table from the csv-dump")
+    parser.add_argument("input", help="The CSV input file (wikidata triple)",
+                        type=CompressedFileType('rb'))
+    parser.add_argument("output",
+                        help="The CSV output file (default=sys.stdout)",
+                        default=sys.stdout, nargs='?',
```

Comment:
Each key om a separate line, please.

Reply:
As I said, python is not like this (you can check other python codebases, like pywikibot or ores)

Reply:
How is it forbidden to put individual arguments on individual lines?

Reply:
It won't give out syntax error and when there are long arguments, it's the best way to do so but because arguments have name in python it's not needed like php and thus no one does it (and let's keep it consistent between other python libraries than our php libraries.